README.md

```diff
@@ -206,7 +155,10 @@ To avoid having to manually install your plugin, build and deploy your plugin us
 
 If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying your plugin. After configuring it, just run:
 
+## License
```

Comment:
Do we need that section?

Reply:
![image](https://github.com/mattermost/mattermost-plugin-circleci/assets/102587100/3629674b-ca53-4419-a759-6451e1c19883)

In the task it was mentioned to follow the outline and add missing section headings and License section was there in outline.

Reply:
You're right. The one caveat is that an empty heading without any content doesn't add value. In cases where the content did exist, but was missing a section title, the title is needed. In cases where that content didn't already exist, no section title is needed.

Reply:
I shall update it